data-raw/cdc_metadata_creation.R

```diff
@@ -0,0 +1,1341 @@
+cdc_bmiage <- tibble::tribble(
```

Comment:
The way we did this in admiralroche was to create these as user facing exported functions and have the user run them as part of the template to create their own metadata and then feed this into their ADaM derivations. this way we don't need to store the datasets as part of the package. what's your thoughts on this @zdz2101 and @Pierre-Wallet? 
also tagging @millerg23 who leads admiralroche in case he can add any more wisdom to help us here.

Reply:
yes thats correct, users could even select what data they wanted to create.
Something like,
 `cdc_bmiage <- get_cdc_data(type = "bmiage")`
 where function get_cdc_data holds the data (created by same `tribble` statements you already have)

Reply:
@zdz2101 we seem to have lost the tribble statements now which were the core part of allowing the user to recreate the metadata? am i missing something?